server/Arcadia.Assistant.Calendar/SickLeave/EmployeeSickLeaveActor.cs

```diff
@@ -176,6 +192,14 @@ protected override bool IsStatusTransitionAllowed(string oldCalendarEventStatus,
                 && (newCalendarEventStatus != SickLeaveStatuses.Approved);
         }
 
+        protected override bool IsDatesChangedAllowed(CalendarEvent oldEvent, CalendarEvent newEvent)
+        {
+            var approvals = this.ApprovalsByEvent[oldEvent.EventId];
+
+            return oldEvent.IsPending && approvals.Count == 0 ||
```

Comment:
well, that needs to be discussed. Put a TODO here, I'm not sure if its right

Reply:
If later we decide to keep as is - these TODOs will stay forever, nobody will remove them :)

Reply:
We can remark them in separate issue as to be discussed later.

Reply:
Reverted.